modules/monitoring-disabling-the-local-alertmanager.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="disabling_the_local_alertmanager_{context}"]
+= Disabling the local Alertmanager
+
+A local Alertmanager that routes alerts from Prometheus instances is enabled by default in the `openshift-monitoring` project of the {product-title} monitoring stack.
+
+If you do not need the local Alertmanager, you can disable it by configuring the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Add `enabled: false` for the `alertmanagerMain` component under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    alertmanagerMain:
+      enabled: false
+----
+
+. Save the file to apply the changes. The Alertmanager instance is disabled automatically when you apply the change.
```

Comment:
I realize this is a boilerplate phrase borrowed from other related topics, but does saving the file really apply the changes? Or must one use an `oc apply` similar to [the one shown here](https://deploy-preview-36300--osdocs.netlify.app/openshift-enterprise/latest/monitoring/configuring-the-monitoring-stack.html#creating-cluster-monitoring-configmap_configuring-the-monitoring-stack)?

Reply:
I'm not sure. @fpetkovski or @juzhao would know the answer, I'm sure. The phrase (or something similar) appears throughout the configuration section of the Monitoring docs.

Reply:
cluster-monitoring-operator would watch and apply the changes
1. if the `cluster-monitoring-config` configmap does not exist, we can use `oc applly` to create and apply the changes
2. if the `cluster-monitoring-config` configmap exists, edit and save `cluster-monitoring-config` configmap would apply the changes